Packs/Jira/Playbooks/playbook-Create_Jira_Issue.yml

```diff
@@ -0,0 +1,524 @@
+        'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "8"
+      "yes":
+      - "3"
+    scriptarguments:
+      brandname:
+        simple: jira-v2
+    results:
+    - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": -240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+system: true
+view: |-
+  {
+    "linkLabelsPosition": {
+      "16_3_yes": 0.52,
+      "9_14_State Polling": 0.52
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1665,
+        "width": 1520,
+        "x": 50,
+        "y": -430
+      }
+    }
+  }
+inputs:
+- key: Summary
+  value: {}
+  required: true
+  description: Set the summary of the ticket.
+  playbookInputQuery:
+- key: Description
+  value: {}
+  required: false
+  description: Set the description of the ticket.
+  playbookInputQuery:
+- key: ProjectKey
+  value: {}
+  required: true
+  description: Jira Project Key
+  playbookInputQuery:
+- key: IssueTypeName
+  value: {}
+  required: true
+  description: TaskName
+  playbookInputQuery:
+- key: SyncTicket
+  value: {}
+  required: false
+  description: "Set the value of the desired sync method with Jira Issue. you can\
+    \ choose one of three options:\n1. StatePolling\n2. Mirror \n3. Blank for none\
+    \ \n\nGenericPolling polls for the state of the ticket and runs until the ticket\
+    \ state is either resolved or closed. \n\nMirror - You can use the Mirror option\
+    \ to perform a full sync with the Jira Ticket. The ticket data is synced automatically\
+    \ between Jira and Cortex xSOAR with the Jira mirror feature.\nIf this option\
+    \ is selected, FieldPolling is true by default. "
+  playbookInputQuery:
+- key: PollingInterval
+  value: {}
+  required: false
+  description: |-
+    Set interval time for the polling to run
+    (In minutes)
+  playbookInputQuery:
+- key: PollingTimeout
+  value: {}
+  required: false
+  description: |-
+    Set the amount of time to poll the status of the ticket before declaring a timeout and resuming the playbook.
+    (In minutes)
+  playbookInputQuery:
+- key: AdditionalPollingCommandName
+  value: {}
+  required: false
+  description: "In this use case, Additional polling commands are relevant when using\
+    \ StatePolling, and there is more than one Jira instance. It will specify the\
+    \ polling command to use a specific instance to run on. \nIf so, please add \"\
+    Using\" to the value. \nThe playbook will then take the instance name as the instance\
+    \ to use. "
+  playbookInputQuery:
+- key: InstanceName
+  value: {}
+  required: false
+  description: |
+    Set the Jira Instance that will be used for mirroring/running polling commands.
```

Comment:
Consider adding
The instance name can be found in the integrations setting page
For example
jira-v2_instance_1